linchpin/provision/roles/azure/tasks/provision_azure_withApi.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Use Azure Api to manage resources
+  azure_rm_resource:
+    client_id: "{{ auth_var['client_id'] | default(omit) }}"
+    tenant: "{{ auth_var['tenant'] | default(omit) }}"
+    secret: "{{ auth_var['secret'] | default(omit) }}"
+    subscription_id: "{{ auth_var['subscription_id'] | default(omit) }}"
+    resource_group: "{{res_def['resource_group']|default(omit)}}"
+    provider: compute
+    resource_type: "{{res_def['resource_type']|default('virtualmachinescalesets')}}"
+    resource_name: "{{res_def['resource_name']|default("myVmss")}}"
+    api_version: "{{res_def['api_version']|default("2019-03-01")}}"
+    body: { body }
```

Comment:
Isn't this file is a leftover of when the role was called 'azure_withApi' and should be removed?

Reply:
yes it is